docs/gwcs/ifu.rst

```diff
@@ -36,6 +36,16 @@ First, import the usual packages.
   >>> from gwcs import wcs, selector
   >>> from gwcs import coordinate_frames as cf
 
+Next, create the appropriate mapper object corresponding to the figure above:
+
+  >>> y, x = np.mgrid[:1000, :500]
+  >>> fmask = (((-x + 0.01 * y + 0.00002 * y**2)/ 500) * 13 - 0.5) + 14
+  >>> mask = fmask.astype(np.int8)
+  >>> mask[(mask % 2) == 1] = 0
+  >>> mask[mask > 13] = 0
+  >>> mask = mask // 2
+  >>> labelmapper = selector.LabelMapperArray(mask)
+
```

Comment:
Honestly, this example does not help me understand how to build an "appropriate mapper", what goes into it, etc. It just looks like a random set of numbers and operations on arrays. Why not add `mask = mask**2` at the end (for example)? That is, I think this needs a detailed explanation. What is `fmask` and how did you derive that equation looking at the image?

Reply:
I was working off of the original example, which did not explain how that object was created (since it just read it from an ASDF file). But it is not a bad idea to explain what this object is for. Part of your comment seems more directed to the details of how this map was generated. For that I'm not sure it is important to explain in detail. After all, there is no real image in this case (nor was there in the original example). Or are you referring to the image of the mask? That was generated from the mask rather than the other way around. The idea was to illustrate how to use this sort of thing when discontinuous areas of the image use different WCS models.
I have added a comment instructing the viewer to ignore the details of the mask construction as they are completely irrelevant to what is being illustrated. I'm not sure what the `mask**2` comment refers to.

Reply:
> ... instructing the viewer to ignore the details of the mask construction as they are completely irrelevant to what is being illustrated.

I am just puzzled but how `mask` is used in `labelmapper = selector.LabelMapperArray(mask)` which is later used in `selector.RegionsSelector()`. Fundamentally, it is not clear to me what role `mask` plays in this or how user should create a `mask` (even if it is irrelevant in this particular example; well, if it is irrelevant, why not make `mask=np.ones(...)`?)